lightning/src/util/events.rs

```diff
@@ -135,6 +135,21 @@ pub enum Event {
 		/// The outputs which you should store as spendable by you.
 		outputs: Vec<SpendableOutputDescriptor>,
 	},
+	/// This event is generated when a payment has been successfully forwarded through us and a
+	/// forwarding fee earned.
+	PaymentForwarded {
+		/// The fee, in milli-satoshis, which was earned as a result of the payment.
+		///
+		/// Note that if we force-closed a channel while a forwarded HTLC was pending, the amount
+		/// the next hop claimed may have been rounded down, causing a higher fee than expected.
```

Comment:
What's the rational for the rouding down ? Do you mean an onchain or off-chain claim, I don't see the case you're mentioning...

Reply:
This is referring to when the outbound forward was claimed on-chain (and then we claimed the backwards link off-chain). Any suggestions for how to phrase this better?

Reply:
Hmmm, "Note that if we force-closed a channel while a forwarded HTLC was pending, the amount the next hop claimed onchain may have been rounded down compare to the backward off-chain HTLC claimed by us, therefore presenting a higher fee than expected" ?

Reply:
Hmm, is that more clear? That just seems like more text to say something similar.

Reply:
Maybe I can just add a "rounded down to the nearest whole satoshi" to make it clear why it's rounded down? I want to avoid the sentence getting too long though.

Reply:
I like "rounded down to the nearest whole satoshi"

Reply:
Well IMHO the comment enhancement is highlighting the comparison, like _rounded down_ compare to _what_ ? Though not a big deal